internal-services/catalog/create-advisory-task.yaml

```diff
@@ -157,6 +157,9 @@ spec:
           /home/utils/apply_template.py -o $ADVISORY_FILEPATH --data "$DATA" \
             --template /home/templates/advisory.yaml.jinja
 
+          # Ensure the created advisory file passes the advisory schema
+          check-jsonschema --schemafile schema/advisory.json "$ADVISORY_FILEPATH"
```

Comment:
How does this does? Where is the schema directory and file coming from? It's definitely not in our utils image: https://github.com/konflux-ci/release-service-utils/blob/main/Dockerfile

Reply:
It's a relative path. It's in the advisory repo. Just like `ADVISORY_DIR="data/advisories/$(params.origin)/${YEAR}/${ADVISORY_NUM}"`

Reply:
Ah, I see, thanks.